app/controllers/application_controller/ci_processing.rb

```diff
@@ -280,11 +280,13 @@ def process_objects(objs, task, display_name = nil)
 
     display_name ||= task.titleize
     case klass_str
-    when 'OrchestrationStack', 'Service', 'CloudObjectStoreContainer', 'CloudObjectStoreObject'
-      objs, _objs_out_reg = filter_ids_in_region(objs, klass.to_s)
+    when 'CloudObjectStoreContainer', 'CloudObjectStoreObject'
+      objs, _objs_out_reg = filter_ids_in_region(objs, klass_str)
     when 'VmOrTemplate'
```

Comment:
Hi @yrudman Why would the condition be different for VM's here?

Reply:
@tinaafitz 
There are several reasons to add separate `when`:
- `Service` would have only one available for centralized Administration task in that method - `retire_now` but `Vm` has much longer list of tasks available for Centralized administration and it would looks confusing to put `Vm` and `Service` together
- value in case condition and parameter passed to `filter_in_region` are different strings for Vm

Reply:
Thanks @yrudman.